docs/tutorials/proposals/update-spot-market.md

```diff
@@ -0,0 +1,313 @@
+
+In addition to the parameters you want to change, you must include all existing editable parameters from the original new market proposal, even if they are not being changed.
+
+### Locked fields
+
+The following fields cannot be changed. They are not included in the [proposal templates](#templates-and-submitting).
+
+- `sizeDecimalPlaces`
+- `priceDecimalPlaces`
+- `baseAsset`
+- `quoteAsset`
+
+### Thresholds
+
+Note that some network parameters may differ, such as the limits on how long the voting period can last, as follows.
+
+| Field                 | Description           |
+| --------------------- | -------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| `closingTimestamp`    | Timestamp (Unix time in seconds) when voting closes for this proposal. The chosen time must be between <NetworkParameter frontMatter={frontMatter} param="governance.proposal.updateMarket.minClose" hideName={true} /> and <NetworkParameter frontMatter={frontMatter} param="governance.proposal.updateMarket.maxClose" hideName={true} /> after the proposal submission time. (int64 as string) |
+| `enactmentTimestamp ` | Timestamp (Unix time in seconds) when proposal gets enacted (if passed). The chosen time must be between <NetworkParameter frontMatter={frontMatter} param="governance.proposal.updateMarket.minEnact" hideName={true} /> and <NetworkParameter frontMatter={frontMatter} param="governance.proposal.updateMarket.maxEnact" hideName={true} /> after `closingTimestamp`. (int64 as string)         |
+
+## Submitting proposals in a batch
+
+<Batch />
+
+## Templates and submitting
+In the tabs below you'll see:
+
+* Annotated example describing what each field is for
+* JSON example that can be submitted with the [governance dApp ↗](https://governance.fairground.wtf/proposals/propose/raw)
+* Command line examples for different operating systems
+
+**Replace the example data with the relevant details before submitting.**
+
+<Tabs groupId="updateSpotMarket">
+  <TabItem value="json" label="Governance dApp (JSON)">
+  	<JSONInstructions />
+
+```json
+{
+"proposalSubmission": {
+  "rationale": {
+    "description": "Propose a change to the spot market for Bitcoin and USDT",
+    "title": "Change Bitcoin/USDT spot market"
+  },
+  "terms": {
+    "closingTimestamp": "1683626590",
+    "enactmentTimestamp": "1683626600",
+    "updateSpotMarket": {
+    "marketId": "123",
+      "changes": {
+          "instrument": {
+          "code": "BTC/USDT",
+          "name": "BTC/USDT"
+          },
+        "tickSize": "10",        
```

Comment:
```suggestion
      "changes": {
        "tickSize": "10",        
```

[decent.markets](https://decent.markets/tx/#ewoicHJvcG9zYWxTdWJtaXNzaW9uIjogewogICJyYXRpb25hbGUiOiB7CiAgICAiZGVzY3JpcHRpb24iOiAiUHJvcG9zZSBhIGNoYW5nZSB0byB0aGUgc3BvdCBtYXJrZXQgZm9yIEJpdGNvaW4gYW5kIFVTRFQiLAogICAgInRpdGxlIjogIkNoYW5nZSBCaXRjb2luL1VTRFQgc3BvdCBtYXJrZXQiCiAgfSwKICAidGVybXMiOiB7CiAgICAiY2xvc2luZ1RpbWVzdGFtcCI6ICIxNjgzNjI2NTkwIiwKICAgICJlbmFjdG1lbnRUaW1lc3RhbXAiOiAiMTY4MzYyNjYwMCIsCiAgICAidXBkYXRlU3BvdE1hcmtldCI6IHsKICAgICJtYXJrZXRJZCI6ICIxMjMiLAogICAgICAiY2hhbmdlcyI6IHsKICAgICAgICAgICJpbnN0cnVtZW50IjogewogICAgICAgICAgImNvZGUiOiAiQlRDL1VTRFQiLAogICAgICAgICAgIm5hbWUiOiAiQlRDL1VTRFQiCiAgICAgICAgICB9LAogICAgICAgICJ0aWNrU2l6ZSI6ICIxMCIsICAgICAgICAKICAgICAgICAibWV0YWRhdGEiOiBbCiAgICAgICAgICAiYmFzZTpCVEMiLAogICAgICAgICAgInF1b3RlOlVTRFQiLAogICAgICAgICAgImNsYXNzOmZ4L2NyeXB0byIsCiAgICAgICAgICAic3BvdCIsCiAgICAgICAgICAic2VjdG9yOmRlZmkiCiAgICAgICAgXSwKICAgICAgICAicHJpY2VNb25pdG9yaW5nUGFyYW1ldGVycyI6IHsKICAgICAgICAgICJ0cmlnZ2VycyI6IFsKICAgICAgICAgICAgewogICAgICAgICAgICAgICJob3Jpem9uIjogIjIxNjAwIiwKICAgICAgICAgICAgICAicHJvYmFiaWxpdHkiOiAiMC45OTk5OTk5IiwKICAgICAgICAgICAgICAiYXVjdGlvbkV4dGVuc2lvbiI6ICI4NjQwMCIKICAgICAgICAgICAgfSwKICAgICAgICAgICAgewogICAgICAgICAgICAgICJob3Jpem9uIjogIjQzMjAiLAogICAgICAgICAgICAgICJwcm9iYWJpbGl0eSI6ICIwLjk5OTk5OTkiLAogICAgICAgICAgICAgICJhdWN0aW9uRXh0ZW5zaW9uIjogIjM2MDAiCiAgICAgICAgICAgIH0sCiAgICAgICAgICAgIHsKICAgICAgICAgICAgICAiaG9yaXpvbiI6ICIxNDQwIiwKICAgICAgICAgICAgICAicHJvYmFiaWxpdHkiOiAiMC45OTk5OTk5IiwKICAgICAgICAgICAgICAiYXVjdGlvbkV4dGVuc2lvbiI6ICIxODAwIgogICAgICAgICAgICB9LAogICAgICAgICAgICB7CiAgICAgICAgICAgICAgImhvcml6b24iOiAiMzYwIiwKICAgICAgICAgICAgICAicHJvYmFiaWxpdHkiOiAiMC45OTk5OTk5IiwKICAgICAgICAgICAgICAiYXVjdGlvbkV4dGVuc2lvbiI6ICIzMDAiCiAgICAgICAgICAgIH0KICAgICAgICAgIF0KICAgICAgICB9LAogICAgICAgICJ0YXJnZXRTdGFrZVBhcmFtZXRlcnMiOiB7CiAgICAgICAgICAidGltZVdpbmRvdyI6ICIzNjAwIiwKICAgICAgICAgICJzY2FsaW5nRmFjdG9yIjogMC4wNQogICAgICAgIH0sCiAgICAgICAgInNsYVBhcmFtcyI6IHsKICAgICAgICAgICJwcmljZVJhbmdlIjogIjAuMDMiLAogICAgICAgICAgImNvbW1pdG1lbnRNaW5UaW1lRnJhY3Rpb24iOiAiMC43NSIsCiAgICAgICAgICAicGVyZm9ybWFuY2VIeXN0ZXJlc2lzRXBvY2hzIjogIjEiLAogICAgICAgICAgInNsYUNvbXBldGl0aW9uRmFjdG9yIjogIjAuOCIKICAgICAgICB9LAogICAgICAgICJsaXF1aWRpdHlGZWVTZXR0aW5ncyI6IHsKICAgICAgICAgICJtZXRob2QiOiAiTUVUSE9EX01BUkdJTkFMX0NPU1QiCiAgICAgICAgfSwKICAgICAgICAibG9nTm9ybWFsIjogewogICAgICAgICAgInJpc2tBdmVyc2lvblBhcmFtZXRlciI6IDFlLTA2LAogICAgICAgICAgInRhdSI6IDMuOTk1ZS0wNiwKICAgICAgICAgICJwYXJhbXMiOiB7CiAgICAgICAgICAgICJzaWdtYSI6IDEuMAogICAgICAgICAgfQogICAgICAgIH0KICAgICAgfQogICAgfQogICB9CiB9Cn0=) says we should remove this field. It's probably ignored if it's invalid, but we might as well make it correct. The rest is all valid.

Reply:
Confirmed with @candida-d, this is valid as-is for the upcoming release. decent.markets is not up to date.